exonum-java-binding/launcher-plugins/tests/test_data/proto_sources/custom_message_name.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.examples.timestamping;
+
+message InitialConfig {
+    // Corresponds to https://developers.google.com/protocol-buffers/docs/encoding#strings
```

Comment:
What does correspond to string encoding?

2 is a field tag and has nothing to do with the on-the wire encoding (the type defines the wire encoding, the tag — identifies the field): https://developers.google.com/protocol-buffers/docs/proto3#assigning-field-numbers

Reply:
I really can't understand why the `2` became such an important detail. It is used just to use the same binary representation as in the example under the provided link.

Changed to `1` and removed links to protobuf docs.